app/assets/scripts/components/spotlight/single/index.js

```diff
@@ -289,11 +307,29 @@ function mapStateToProps (state, props) {
       } else {
         return {
           ...l,
+          domain: layerProps[l.type] ? layerProps[l.type].domain : l.domain,
```

Comment:
For any other site than `la` and `sf` this throws an error breaking the site

Reply:
thanks!